support-frontend/assets/pages/weekly-subscription-checkout/components/weeklyCheckoutFormGifting.tsx

```diff
@@ -180,17 +148,24 @@ function mapDispatchToProps() {
 	};
 }
 
+const connector = connect(mapStateToProps, mapDispatchToProps());
+
+// ----- Types ----- //
+type PropTypes = ConnectedProps<typeof connector>;
```

Comment:
nice!

Reply:
I guess I'm slightly mixed on this one - it's nice to reduce code, but having an explicit definition of prop types is also quite nice for quickly checking what type everything is

Reply:
@TomPretty I see where you're coming from, although with my IDE (Visual Studio) it's possible to view the properties and types of `PropTypes` still.